### PR TITLE
Part 1

### PR DESCRIPTION
**Pull request by AB-HardWorkBeatsNaturalTalent**
added nouns, actions etc.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,84 +52,112 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
-
+//  Noun: Animal
+//  action 1: the animal drinks
+a.drink()
+
+//  action 2: the animal eats
+a.eat()
+//  action 3: the animal replicates
+a.replicate()
```
Comment (matkatmusic) [inline main.cpp]: the noun is `animal`, not `a`
you forgot to add `;` at the end of each pseudo-code expression
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -52,84 +52,112 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
+//  action 1: university covers up a scandal
+u.coverUpScandal()
 
-//  action 2:
+//  action 2: university accepts bribe from big pharma
+u.acceptBribeFromBigPharma()
 
-//  action 3:
+//  action 3: university honors the corporate elite under guise of philanthropy
+u.honorCorporateEliteUnderGuiseOfPhilanthropy()
```
Comment (matkatmusic) [inline main.cpp]: write out the name of the noun in the pseudo-code, using the naming convention described in the coding standard written in the Readme.MD file.

you forgot to add `;` at the end of each pseudo-code expression
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
you forgot to add `;` at the end of each pseudo-code expression
- [x] revise

File: main.cpp (hunk 1 above)
Comment (AB-HardWorkBeatsNaturalTalent) [reply main.cpp]: added nounds and semicolons in new commits.

File: main.cpp (hunk 2 above)
Comment (AB-HardWorkBeatsNaturalTalent) [reply main.cpp]: added nouns and semicolons

**Review by AB-HardWorkBeatsNaturalTalent — COMMENTED** [review]
I made the changes you requested.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**